configs/projects/puppet-agent.rb

```diff
@@ -6,6 +6,7 @@
     proj.setting(:company_name, "Puppet Labs")
     proj.setting(:company_id, "PuppetLabs")
     proj.setting(:common_product_id, "PuppetInstaller")
+    proj.setting(:PuppetServiceName, "puppet")
```

Comment:
Rather than using camel case here, I'd prefer we keep it in the ruby convention with underscores. Not a blocker, just a personal preference. If that opinion isn't shared, I'm not going to fight for it.